external/lfric_infrastructure/preprocessed/PSYKE/dino_mod.f90

```diff
@@ -0,0 +1,310 @@
+
```

Comment:
Is this actually used somewhere?

Reply:
I hope not ;)
I'd think it's up the LFRic to remove this if not required. Starting to manually exclude files feels annoying). But I am happy to remove it if you want.